### PR TITLE
Fix for videos starting muted in Chrome

### PR DESCRIPTION
**Pull request by ellamenop**
Instead of starting the video + muting it to begin buffering, start the video + pause it.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -62,11 +62,11 @@ function onResize(player) {
 }
 
 // It seems there's no explicit way to load a video,
-// so instead we mute the video and start playing it immediately.
+// so instead we start playing the video, then pause immediately
```
Comment (kofigumbs) [inline script.js:65]: 👍🏾this makes sense to me

File: script.js [hunk 2]
```diff
@@ -88,15 +88,12 @@ function onMidi(player, data, time) {
     // NOTE OFF
     const note = hold(data[1], false);
     controls[note].classList.remove("pressed");
-    if (!Object.values(holding).includes(true))
-      player.mute();
```
Comment (kofigumbs) [inline script.js]: My thinking here is that the audio plays for as long as you're holding the note—kinda simulating a "release". If you delete this, then the video/audio will keep playing until the end right?

Comment (ellamenop) [reply script.js]: Oh I gotchu, my bad! Un-deleted this in the latest commit, but changed `mute()` to `pause()` in keeping w/ the other changes 👍

**Comment by kofigumbs** [comment]
For some reason, the video doesn't start buffering for me here until I press a key. I think we could listen to the play state change event and pause the video as soon as it actually starts.

This whole thing made me realize I never asked something: was the muting approach messing up something with your workflow?

**Comment by ellamenop** [comment]
Okay, I feel like a dum dum. Turns out I just wasn't holding the keys down, which is why it wasn't making any sound 🤦‍♂️

Thanks for helping me get to the bottom of this! The current `master` branch works great.

**Comment by kofigumbs** [comment]
🎉(also, there are literally no instructions, so it's on me)

**Pull request merged**